backend/src/model/route.py

```diff
@@ -0,0 +1,11 @@
+from sqlalchemy import ForeignKeyConstraint
+from sqlalchemy.orm import Mapped, mapped_column
+from src.db import Base
+
+
+class RouteModel(Base):
+    __tablename__ = "routes"
+    id: Mapped[int] = mapped_column(
+        primary_key=True, autoincrement=True, nullable=False
+    )
+    name: Mapped[str] = mapped_column(unique=True)
```

Comment:
Is there a reason this isn't marked `nullable=false`

Reply:
Copilot hallucination